sessions/class-01/exercise/start.js

```diff
@@ -22,7 +33,10 @@ export function safelyConvertToNumber(possiblyNum) {}
  * @param {number} decimal Decimal precision.
  * @returns {number} Formatted currency value.
  */
-export function formatCurrency(num, decimal) {}
+export function formatCurrency(num, decimal) {
+    let number = safelyConvertToNumber(num).toFixed(decimal);
+    return Number(number);
```

Comment:
Good job on converting the return type to number. Again use `const` instead of `let`, since not reassigning. Also we can skip the variable declaration altogether

```js
return +safelyConvertToNumber(num).toFixed(decimal);
```